src/test/ui/lint/lint-non-ascii-idents.rs

```diff
@@ -0,0 +1,11 @@
+#![feature(non_ascii_idents)]
```

Comment:
Could you move this into a folder under `ui/lint/` dedicated to RFC 2457 (since there are more lints to add later...)